PWGHF/HFC/Utils/utilsCorrelations.h

```diff
@@ -29,22 +30,22 @@ enum Region {
 template <typename T>
 Region getRegion(T deltaPhi)
 {
-  if (std::abs(deltaPhi) < o2::constants::math::PI / 3.) {
+  if (std::abs(deltaPhi) < o2::constants::math::PIThird) {
     return Toward;
-  } else if (deltaPhi > 2. * o2::constants::math::PI / 3. && deltaPhi < 4. * o2::constants::math::PI / 3.) {
+  } else if (deltaPhi > 2. * o2::constants::math::PIThird && deltaPhi < 4. * o2::constants::math::PIThird) {
     return Away;
   } else {
     return Transverse;
   }
 }
 
 // Find Leading Particle
-template <typename TTracks>
-int findLeadingParticle(TTracks const& tracks)
+template <typename TTracks, typename T1, typename T2>
+int findLeadingParticle(TTracks const& tracks, T1 const dcaXYTrackMax, T2 const dcaZTrackMax)
 {
   auto leadingParticle = tracks.begin();
   for (auto const& track : tracks) {
-    if (std::abs(track.dcaXY()) >= 1. || std::abs(track.dcaZ()) >= 1.) {
+    if (std::abs(track.dcaXY()) >= dcaXYTrackMax.value || std::abs(track.dcaZ()) >= dcaZTrackMax.value) {
```

Comment:
The `.value` should not be called inside the helper function. It should be called inside the task when calling the function.